src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -697,10 +710,14 @@ public DeleteBackupResult deleteBackup(DeleteBackupRequest deleteBackupRequest)
         if (backupManager.isPresent()) {
```

Comment:
validate if mt context is not specified here (no tenant specific delete backup supported yet)